docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Apollo Router Quickstart
+subtitle: Run a router binary
+description: This quickstart tutorial walks you through installing an Apollo Router binary, running it with an example supergraph schema and a YAML configuration file, and making test queries with Apollo Sandbox.
+---
+
+import ElasticNotice from "../../shared/elastic-notice.mdx";
+
+Hello! Let's run Apollo Router for the first time, using the simple scenario of developing and deploying locally.
+
+In this guide, you will:
+
+- Download and install a router binary.
+- Download a supergraph schema.
+- Create a router YAML configuration file.
+- Run the router in development mode.
+- Make a query to the running router.
+
+## 1. Install router binary
+
+Let's start by downloading and installing a router binary locally.
+
+1. Download and install the latest version of the router binary with a single command line:
+
+    ```bash showLineNumbers=false
+    curl -sSL https://router.apollo.dev/download/nix/latest | sh
+    ```
+
+    <ExpansionPanel title="Download a specific router version">
+
+    To download and install a specific version of router, set the version in the download URL path.
+
```

Comment:
Until we fix spacing in `<ExpansionPanel>`s, We need to include `<br>s` to avoid squishy text
<img width="768" alt="image" src="https://github.com/user-attachments/assets/9717c580-6a70-4883-a53f-682a5a552a63" />